### PR TITLE
refactor: Linter fix

### PR DESCRIPTION
**Pull request by guacamole**
- Made code fixes as to comply with golangci linter

### CODE REVIEW

**Comment by jay-dee7** [comment]
@guacamole can you please rebase this branch from master?

File: registry/v2/blobs.go [hunk 1]
```diff
@@ -30,35 +30,36 @@ func (b *blobs) errorResponse(code, msg string, detail map[string]interface{}) [
...
-	if !ok {
-		return
-	}
-	delete(b.layers, repo)
-
-	for _, d := range digests {
-		delete(b.contents, d)
-	}
-}
+//func (b *blobs) get(namespace string) (map[string][]byte, bool) {
```
Comment (jay-dee7) [inline registry/v2/blobs.go:33]: @guacamole can we track this via another issue to clean it up?

File: registry/v2/blobs.go [hunk 2]
```diff
@@ -30,35 +30,36 @@ func (b *blobs) errorResponse(code, msg string, detail map[string]interface{}) [
...
+//	digests, ok := b.layers[namespace]
```
Comment (jay-dee7) [inline registry/v2/blobs.go:34]: and this

File: registry/v2/helpers.go [hunk 3]
```diff
@@ -28,24 +27,24 @@ func (r *registry) errorResponse(code, msg string, detail map[string]interface{}
...
-		return "", &RegistryError{
-			Code:    RegistryErrorCodeNameInvalid,
-			Message: "blobs must be attached to a repo",
-			Detail:  map[string]interface{}{},
-		}
-	}
-
-	return elem[len(elem)-1], nil
-}
+//func (r *registry) getDigestFromURI(u *url.URL) (string, *RegistryError) {
```
Comment (jay-dee7) [inline registry/v2/helpers.go:30]: and this too

**Review by jay-dee7 — APPROVED** [review]
LGTM 🥇 thanks for adding this

File: registry/v2/blobs.go (hunk 1 above)
Comment (guacamole) [reply registry/v2/blobs.go:33]: being tracked via #42

File: registry/v2/helpers.go (hunk 3 above)
Comment (guacamole) [reply registry/v2/helpers.go:30]: being traced here #42

**Pull request merged**